src/java/main/org/apache/zookeeper/server/persistence/FileTxnSnapLog.java

```diff
@@ -132,6 +137,9 @@ public FileTxnSnapLog(File dataDir, File snapDir) throws IOException {
 
         txnLog = new FileTxnLog(this.dataDir);
         snapLog = new FileSnap(this.snapDir);
+
+        autoCreateDB = Boolean.parseBoolean(System.getProperty(ZOOKEEPER_DB_AUTOCREATE,
```

Comment:
>> Is that in accord with Zookeeper style?

I see - I thought the new property was not end user facing since there is no associated documents added here. Since the property "zookeeper.db.autocreate" is exposed to user some doc could be added to ZooKeeperAdmin.html (similarly like how the existing "zookeeper.datadir.autocreate" is documented there) to describe the motivation / usage of the property.